### PR TITLE
Add Basque (eu) locale (closes #861)

### LINKED ISSUE #861: Basque (eu) language support
We're starting to work on Basque (euskara) language support. It is an official language in Spain like catalan, spanish and galician.

-ISO-639-2: eus
-ISO-639-1: eu

### PR DESCRIPTION
**Pull request by JacobSoderblom**
Support for Basque will be added with this PR and will close #861

### CODE REVIEW

File: src/locale/eu/_lib/formatDistance/index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    one: '1 ordu',
+    other: '{{count}} ordu'
+  },
+
+  xDays: {
+    one: '1 egun',
+    other: '{{count}} egun'
+  },
+
+  aboutXMonths: {
```
Comment (kossnocorp) [inline src/locale/eu/_lib/formatDistance/index.js:39]: Please add [recently introduced `aboutXWeeks` and `xWeeks`](https://github.com/date-fns/date-fns/pull/1671)

Comment (JacobSoderblom) [reply src/locale/eu/_lib/formatDistance/index.js:39]: Sure! I guess that I can find an example from the other locales?

Comment (kossnocorp) [reply src/locale/eu/_lib/formatDistance/index.js:39]: Yes, essentially, you need to add tokens similar to months, but for weeks.

Comment (JacobSoderblom) [reply src/locale/eu/_lib/formatDistance/index.js:49]: aboutXWeeks & xWeeks are now added

**Comment by JacobSoderblom** [comment]
@kossnocorp please check again when you have time

**Comment by LinusTrulsson** [comment]
@kossnocorp Hello! I'm wondering when this can be approved and merged? :)

**Comment by kossnocorp** [comment]
@LinusTrulsson sorry I can't give you any timeline, as you might imagine I have other responsibilities besides date-fns, but that's on the top of my list.

**Comment by kossnocorp** [comment]
I encourage you to copy the locale source code to your project and use it from there for the time being if it's something that blocks you.

**Pull request merged**